Multisig/UI/Settings/OwnerKeyManagement/AddOwnerKey/EnterOwnerAddressViewController.swift

```diff
@@ -66,7 +66,7 @@ class EnterOwnerAddressViewController: UIViewController {
                 info: nil,
                 chainId: self.safe.chain!.id!)
 
-            self.addressField.setAddress(address, label: resolvedName)
+            self.addressField.setAddress(address, label: resolvedName, prefix: self.safe.chain!.shortName)
```

Comment:
For safer code we can remove the force unwrapping so we use 
```suggestion
            self.addressField.setAddress(address, label: resolvedName, prefix: self.safe.chain?.shortName)
```

Reply:
Good catch! Thanks!